src/components/ItemFileInput.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useRef, useState } from "react";
+import { Flex } from "../styles/styled";
+import {
+  FileImage,
+  FileInput,
+  FileLabel,
+  FileWrap,
+  Label,
+} from "../styles/styled/Additem";
+import PlusIcon from "../assets/ic_plus@3x.png";
+import CancelBlueLogo from "../assets/ic_blue_X@3x.png";
+import styled from "styled-components";
+
+const Cancel = styled.img`
+  position: absolute;
+  top: 18px;
+  right: 18px;
+  width: 22px;
+  height: 24px;
+  cursor: pointer;
+  @media (max-width: 1199px) {
+    top: 12px;
+    right: 12px;
+  }
+`;
+
+function ItemFileInput({ value, name, onChange }) {
+  const [preview, setPreview] = useState();
+  const fileInputRef = useRef();
+
+  const handleChange = (e) => {
+    const nextFile = e.target.files[0];
+    onChange(name, nextFile);
+  };
+
+  const onCancel = (value) => {
```

Comment:
```suggestion
  const onCancel = (value) => {
```
이미지를 삭제하는 함수면 삭제라는 의미에 remove가 어울릴것같네요